screens/ProfileScreen.tsx

```diff
@@ -1,23 +1,34 @@
 import * as React from 'react';
-import Icon from '../assets/svg-icons/settings.svg';
-import { StyleSheet } from 'react-native';
+import PlusIcon from '../assets/svg-icons/plus-circle.svg';
+import SettingsIcon from '../assets/svg-icons/settings.svg';
+import SearchIcon from '../assets/svg-icons/search.svg';
+import { StyleSheet, TouchableOpacity } from 'react-native';
 import { Text, View } from '../components/Themed';
 
+/* Constants for in TSX inherited properties */
+const pressedOpacity = 0.3;
+
 export default function TabTwoScreen() {
   return (
     <View style={styles.container}>
         <View style={styles.upperContainer}>
-            <View style={styles.profileBubbleContainer}>
-              <View style={styles.profileBubble} />
-            </View>
+            <TouchableOpacity activeOpacity={pressedOpacity}>
+              <SettingsIcon />
+            </TouchableOpacity>
             <View style={styles.profileTextContainer}>
-                <Text style={styles.profileNameText}>ravina</Text>
-                <Text style={styles.profileBioText}>selling cool stuff!</Text>
-            </View>
-            <View>
-                <Icon width={26} height={26} />
+                <View style={styles.profileBubble} />
             </View>
+            <TouchableOpacity activeOpacity={pressedOpacity}>
+              <SearchIcon />
+            </TouchableOpacity>
         </View>
+        <View style={styles.profileTextContainer}>
+            <Text style={styles.profileNameText}>Sergio Pablo Diaz</Text>
+            <Text style={styles.profileBioText}>Junior in the college of engineering. Selling meth, pot, and crack cocaine. This some more sample text to see whether or not the padding disappears.</Text>
+        </View>
+        <TouchableOpacity style={styles.plusButton}>
+          <PlusIcon width={36} height={36} />
```

Comment:
I forget, do you need the curly brackets around int literals?

Reply:
yes you do